about/people/index.md

```diff
@@ -10,6 +10,7 @@ For more information about who is arXiv please see:
 
 - [Staff Leadership Team](leadership_team)
 - [Moderators](/moderators/)
+- [Developers](developers)
```

Comment:
How about Developers -> Volunteer Developers

Reply:
Fixed in 1b6c7f5